src/components/personal-info.jsx

```diff
@@ -0,0 +1,35 @@
+import React, {useState} from "react";
+
+function PersonalInfo() {
+  const [userInfo, setUserInfo] = useState({
+    name: "",
+    number: "",
+    email: ""
+  })
+
+  const handleChange = (e) => {
+    setUserInfo({...userInfo, [e.target.name]: e.target.value})
+  }
+
+  return (
+
+    <form className="resumeSection">
```

Comment:
```suggestion
    <form onSubmit={handleSubmit} className="resumeSection">
```

Finally, we assign the callback to the onSubmit event listener